Testing/integration/test_sync_changes.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -x
+
+bazel run //Testing/integration:install_profile -- Testing/integration/configs/default.mobileconfig
+
+# Reset moroz to default config
+killall moroz
+/tmp/moroz -configs="$GITHUB_WORKSPACE/Testing/integration/configs/moroz_default/global.toml" -use-tls=false &
```

Comment:
In the future, we should probably have test with TLS enabled, to make sure we're testing all paths.